docs/tables/pagerduty_incident_log.md

```diff
@@ -0,0 +1,54 @@
+# Table: pagerduty_incident_log
+
+Retrieves the record of all log entries for the specified incident.
+
+## Examples
+
+### List all activities on specified incident in last 3 days
+
+```sql
+select
+  id,
+  incident_id,
+  created_at,
+  agent ->> 'summary' as created_by
+from
+  pagerduty_incident_log
+where
+  incident_id = 'Q0FH5K82AJ101C'
+  and created_at > now() - interval '3 days';
+```
+
+### List all activities performed by users
+
+```sql
+select
+  id,
+  incident_id,
+  created_at,
+  jsonb_pretty(channel) as action,
+  agent ->> 'summary' as user_name
+from
+  pagerduty_incident_log
+where
+  incident_id = 'Q0FH5K82AJ101C'
+  and created_at > now() - interval '3 days'
```

Comment:
Is there a reason we limit the incident logs we retrieve to 3 days? How many incident logs/how far back do they usually go for a given incident?

Reply:
@cbruno10 there is no specific reason to limit the date range to 3 days. API doesn't have any specific max. limit set on the date range.

Reply:
@Subhajit97 Can you please remove the `created_at` clauses in this example, and the one below?